lib/id3c/cli/redcap.py

```diff
@@ -494,7 +494,7 @@ def _fetch(self, content: str, parameters: Dict[str, str] = {}, *, format: str =
         # in many cases succeeds with additional attempts.
         # -drr, 7/28/2021
         while retry_count <= max_retry_count:
-            response = requests.post(self.api_url, data=data, headers=headers)
+            response = requests.post(self.api_url, data=data, headers=headers, timeout=300)
```

Comment:
Thanks for adding this in. Excited to see if this helps alleviate some of the persistent issues. One thought - with a timeout, would it be appropriate to retry the request?

Reply:
Based on our discussion at Standup this morning, we'll implement it as-is for now and see how often it happens before deciding whether or not to retry on timeout.